content/dart/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,255 @@
+
+On the example below all variables (myInteger, myDouble, myBoolean, and myString) are declared with an explicit data type (int, double, bool, and String, respectively).
+
+```
+int myInteger = 42;
+double myDouble = 3.14;
+bool myBoolean = true;
+String myString = "Hello, Codecademy!";
+```
+Below are a couple of examples with incorrect declarations of values to variables.
+
+```
+// Non-integer value to an int variable
+int myInteger = "42";
+
+// Omitted the explicit data type for a variable
+var missingType = 3.14;
+
+// Declared a boolean variable but provided a string value
+bool myBoolean = "false";
+
+// Mismatch between declared type and assigned value
+String myString = 123;
+
+```
+
+## Numbers
+
+Numbers are used to hold numeric values. They can be classified as `int`,`num` and `double`.
+
+```
+int wholeNumber = 42;
+double floatingPointNumber = 3.14;
+num inheritedNumber = 7; // num can hold both int and double values
+
+print(wholeNumber);          // Output: 42
+print(floatingPointNumber);  // Output: 3.14
+print(inheritedNumber);      // Output: 7
+
+```
+
+Below is an example with incorrect declarations of values to ``int``, ``num``, and ``double`` data type variables.
+
+```
+int invalidInt = "42"; // Assigning a string to an int
+double missingType = 3; // Assigning an int to a double
+bool booleanNumber = 3.14; // Assigning a double to a bool
+String stringNumber = 123; // Assigning an int to a string
+
+print(invalidInt);       // Incompatible types
+print(missingType);      // Missing type annotation
+print(booleanNumber);    // Incompatible types
+print(stringNumber);     // Incompatible types
+
+```
+
+## Strings
+
+Strings``string`` are used to represent a sequence of characters, and they are embedded in either single'...' or double-quotes "...". 
+
+```
+String greeting = "Double quotes work!";
+String anotherString = 'Single quotes work too!';
+
+print(greeting);  // Output: Double quotes work!
+print(anotherString);  // Output: Single quotes work too!
+
+```
+
+Below is an example with incorrect declarations of values to ``string`` data type variables.
+
+```
+String missingQuotes = Hello, Dart!; // Missing quotes around the string
+
+print(missingQuotes); // Outcome: Error, missing quotes
+
+```
+
+## Booleans
+
+Booleans ``bool`` represents values true and false. 
+
+```
+bool isDartFun = true;
+bool isPythonFun = false;
+
+print(isDartFun); // Output: true
+print(isPythonFun); // Output: false
+
+```
+
+## Symbols
+
+Symbol ``Symbol`` object represents an operator or identifier. Objects are particularly valuable for APIs that identify elements by name. This is crucial for APIs because while minification can alter identifier names, it does not impact the underlying identifier symbols.
```

Comment:
```suggestion
Symbol `Symbol` object represents an operator or identifier. Objects are particularly valuable for APIs that identify elements by name. This is crucial for APIs because while minification can alter identifier names, it does not impact the underlying identifier symbols:
```